docs/source/docs/level-up/intermediate_skills/12_framework_conversion.rst

```diff
@@ -0,0 +1,56 @@
+============================
+Level 12: Framework Conversion
+============================
+
+Datumaro allows seamless conversion of datasets to popular deep learning frameworks, such as PyTorch and TensorFlow.
+This is particularly useful when you are working with a dataset that needs to be used across different frameworks
+without manual reformatting.
+
+Datumaro provides the FrameworkConverter class, which can be used to convert a dataset for various tasks
+like classification, detection, and segmentation.
+
+Supported Tasks
+  - Classification
+  - Multilabel Classification
+  - Detection
+  - Instance Segmentation
+  - Semantic Segmentation
+  - Tabular Data
+
+.. tab-set::
+
+  .. tab-item:: Python
+
+    With the PyTorch framework, you can convert a Datumaro dataset like this:
+
+    .. code-block:: python
+
+      from datumaro.plugins.framework_converter import FrameworkConverter
+      from torchvision import transforms
+
+      transform = transforms.Compose([transforms.ToTensor()])
+      dm_dataset = ...  # Load your dataset here
+
+    First, we have to specify the dataset, subset, and task
+
+    .. code-block:: python
+
+      multi_framework_dataset = FrameworkConverter(dm_dataset, subset="train", task="classification")
+      train_dataset = multi_framework_dataset.to_framework(framework="torch", transform=transform)
+
+    Thourgh this, we convert the dataset to PyTorch format
```

Comment:
Thourgh -> Through, I guess.

Reply:
Fix in [53b3c6f](https://github.com/openvinotoolkit/datumaro/pull/1659/commits/53b3c6f69388137dbb7784721e06c667bae7660d). Thanks for letting me know this.